examples/petsc/bp1.h

```diff
@@ -45,8 +47,10 @@ CEED_QFUNCTION(Mass)(void *ctx, const CeedInt Q,
   const CeedScalar *u = in[0], *rho = in[1];
   CeedScalar *v = out[0];
 
-  for (CeedInt i=0; i<Q; i++) {
+  // Quadrature Point Loop
+  CeedPragmaOMP(simd)
+  for (CeedInt i=0; i<Q; i++)
     v[i] = rho[i] * u[i];
-  }
+
```

Comment:
I left the braces around the loop body for uniformity, though I suppose it does not make a big difference.